packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-block.tsx

```diff
@@ -17,4 +17,5 @@ export default {
     </svg>
   ),
   styles: {},
+  exportedAs: 'block',
```

Comment:
Should we export as "ban", like font awesome?
![image](https://user-images.githubusercontent.com/828692/55172800-d92a7400-5137-11e9-96d5-89bb7e429f3f.png)

Reply:
will do that, thanks!